table.csv

```diff
@@ -108,6 +108,7 @@ garlic64,                       multiaddr,      0x01be,         draft,     I2P b
 garlic32,                       multiaddr,      0x01bf,         draft,     I2P base32 (hashed public key or encoded public key/checksum+optional secret)
 tls,                            multiaddr,      0x01c0,         draft,
 noise,                          multiaddr,      0x01c6,         draft,
+insecure,                       multiaddr,      0x01cb,         draft,
```

Comment:
When dialing a Multiaddr with the `insecure` protocol, which security protocol would be used to exchange the peer IDs? [Plaintext v2](https://github.com/libp2p/specs/tree/master/plaintext)? Why not define this as the `plaintextv2` protocol?

```suggestion
plaintextv2,                       multiaddr,      0x01cb,         draft,
```

Reply:
Yes, plaintextv2. And yeah, that might be a better name.